### PR TITLE
Add some metrics about KVStore

### LINKED ISSUE #8068: Add some more metrics for Raft

### PR DESCRIPTION
**Pull request by CalvinNeo**
### What problem does this PR solve?

Issue Number: close #8068

Problem Summary:

### What is changed and how it works?

We also remove type_exec_compact, since it is very similar to type_passive_flush.

image

image

image

image

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by hongyunyan — APPROVED** [review]
LGTM

File: dbms/src/Common/TiFlashMetrics.h [hunk 1]
```diff
@@ -419,13 +426,20 @@ namespace DB
...
+      F(type_write, {{"type", "write"}}))                                                                                           \
     M(tiflash_raft_region_flush_size,                                                                                               \
       "Bucketed histogram of region flushed size",                                                                                  \
       Histogram,                                                                                                                    \
-      F(type_flushed, {{"type", "flushed"}}, ExpBuckets{32, 2, 16}),                                                                \
-      F(type_unflushed, {{"type", "unflushed"}}, ExpBuckets{32, 2, 16}))                                                            \
+      F(type_flushed, {{"type", "flushed"}}, ExpBuckets{32, 2, 22}),                                                                \
+      F(type_unflushed, {{"type", "unflushed"}}, ExpBuckets{32, 2, 22}))                                                            \
```
Comment (JaySon-Huang) [inline dbms/src/Common/TiFlashMetrics.h:438]: Why we need to increase this bucket size?

File: dbms/src/Common/TiFlashMetrics.h [hunk 2]
```diff
@@ -419,13 +426,20 @@ namespace DB
...
-      F(type_flush_eager_gc, {{"type", "flush_eager_gc"}}),                                                                         \
-      F(type_exec_compact, {{"type", "exec_compact"}}))                                                                             \
+      F(type_flush_eager_gc, {{"type", "flush_eager_gc"}}))                                                                         \
+    M(tiflash_raft_raft_frequent_events_count,                                                                                      \
+      "Raft frequent event counter",                                                                                                \
+      Counter,                                                                                                                      \
+      F(type_write, {{"type", "write"}}))                                                                                           \
     M(tiflash_raft_region_flush_size,                                                                                               \
```
Comment (JaySon-Huang) [inline dbms/src/Common/TiFlashMetrics.h:434]: I think `xxx_bytes` is more suitable than `_size`

File: dbms/src/Common/TiFlashMetrics.h (hunk 1 above)
Comment (CalvinNeo) [reply dbms/src/Common/TiFlashMetrics.h:438]: `region_compact_log_min_bytes` is set to 32MB by default. `32 * 2**16` can't cover.

**Review by JaySon-Huang — APPROVED** [review]

**Pull request merged**